conda/meta.yaml

```diff
@@ -8,28 +8,16 @@ source:
 
 build:
   script: python -m pip install --no-deps --ignore-installed .
-  script_env:
-    - PATH
-    - PYTHONPATH
-    - LD_LIBRARY_PATH
-    - OSPL_HOME
-    - PYTHON_BUILD_VERSION
-    - PYTHON_BUILD_LOCATION
-    - TS_CONFIG_ATTCS_DIR
-    - LSST_DDS_DOMAIN
 
 test:
   requires:
     - ts-conda-build =0.3
-    - ts-externalscripts
-    - ts-standardscripts
```

Comment:
Is there a problem with requiring these? The tests are more complete if we can check that ScriptQueue finds these by default.